structure/deque/deque_test.go

```diff
@@ -0,0 +1,150 @@
+// Tests for the deque package.
+
+package deque
```

Comment:
Do black box testing in the package `deque_test`.

Reply:
The tests are still in the `deque` package? How is this resolved?

Reply:
Hey, sorry about that. I don't get what is asked. 
Cause I was looking at the other test codes and wrote my one accordingly.
Could you clarify please? It would be really helpful

Reply:
You should write the tests under the package `deque_test` instead of `deque` cause you are testing the public API.

https://github.com/TheAlgorithms/Go/blob/679acf08f4b43385e2cb2f13ff0c8a93a6100027/structure/deque/deque_test.go#L3

Reply:
Ah I see, got it now. Thanks for the clarification!